etc/calico-gen-bird6-conf.sh

```diff
@@ -18,7 +18,7 @@ where
   <my-ipv6-address> is the external IPv6 address of the local machine
   <rr-ipv6-address> is the IPv6 address of the route reflector that
       the local BIRD6 should peer with
-  <as-number> is the BGP AS number that the route relector is using.
+  <as-number> is the BGP AS number that the route reflector is using.
```

Comment:
I'd rather we kept the bird6 config generation the same as the IPv4.  Makes sense to get IPv4 tested first, but before we merge, it would be nice to extend the change to IPv6.